triagebot.toml

```diff
@@ -22,6 +23,7 @@ Thanks! <3
 label = "ICEBreaker-LLVM"
 
 [ping.icebreakers-cleanup-crew]
+alias = ["cleanup", "shrink", "reduce", "bisect"]
```

Comment:
I would personally prefer that we avoid adding all the aliases we can think of - is cleanup not enough?

Reply:
There were more that I thought of... ;) ...but these were the ones which I associate the task with most closely, i.e. I associate the actual work more closely with bisecting or shrinking than with "cleanup" (which I also associate with C-cleanup, and that isn't what this is about). Imo having more aliases which aren't misleading means folks are more likely to get the bot to do what they want. (It's good to make the bot fault tolerant, e.g. `cleanups` (with the `s` at the end) would be a good extension I think.)

Reply:
I don't want to try to make the code itself fault tolerant in that respect, but I suppose I can buy this argument. Could you add cleanups and such as well then? r=me with that

Reply:
Added "llvms" and "cleanups". We can add more variations if people make those mistakes. :)